src/FAI3_backend/src/job_management.rs

```diff
@@ -0,0 +1,163 @@
+use candid::Principal;
+
+use crate::types::Job;
+use crate::{JOBS, NEXT_JOB_ID};
+
+#[ic_cdk::update]
+pub fn create_job(model_id: u128) -> u128 {
+    let id = NEXT_JOB_ID.with(|id| {
+        let current_id = *id.borrow().get();
+
+        id.borrow_mut().set(current_id + 1).unwrap();
+
+        current_id
+    });
+
+    let owner_id = ic_cdk::caller();
+
+    let job = Job {
+        id: id,
+        model_id,
+        owner: owner_id,
+        status: "Pending".to_string(),
```

Comment:
I wonder if status should be a constant, or if we should store the possible states as string constants somewhere (probably the second option).

Reply:
@LeonardoRyuta the constants can live in this same file.